.github/workflows/ci-test.yml

```diff
@@ -290,22 +290,6 @@ jobs:
             echo "PREV_TAG=$compat_prev_tag" >> $GITHUB_ENV
             echo "BUILD_PREV=0" >> $GITHUB_ENV
           fi
-      - name: build extra images
```

Comment:
I think we do need to keep this since compatibility test is just disabled but not removed?

Reply:
This specifically does cti-codebuild.sh, which doesn't do forge builds correct?

Reply:
we might need to change cti-codebuild script under land-blocking at line 
```
docker/build-aws.sh --build-all-cti --version $VERSION --addl_tags canary,${ADDL_TAG}
```
to 
```
docker/build-aws.sh --build-forge --version $VERSION --addl_tags canary,${ADDL_TAG}
```

I can fix this in another PR